service/src/main/resources/db/changesets/20230602_schema_extend.yaml

```diff
@@ -1,9 +1,7 @@
 databaseChangeLog:
   - changeSet:
-      id: avoid_postgres_specific_features
+      id: schema_extend
```

Comment:
This changeset file has already been deployed to our test instance (we deploy nightly), so you can't change the name or any of the schema specification for postgres. If you change it to render differently for mariadb/mysql only, then I think that's okay (though I'd want to test to confirm) but changing the postgres rendering will definitely break things.

Reply:
I do not follow.